Modules/Microsoft365DSC/Dependencies/Manifest.psd1

```diff
@@ -48,6 +48,10 @@
             ModuleName      = 'Microsoft.Graph.DeviceManagement.Administration'
             RequiredVersion = '2.0.0'
         },
+        @{
+            ModuleName      = 'Microsoft.Graph.Identity.DirectoryManagement'
```

Comment:
We should be loading both v1 and beta. When Beta is required in at least one resource, then the beta module is what we should be using across the board. Otherwise we end up loading too many functions in scope.

Reply:
I have added v1.0 because of administrative unit resource using v1.0

I think it should be updated to the beta api as mentioned #3450

Reply:
No changes  as AADAdministrativeUnit using v1.0